src/ui/profile_preferences.ui

```diff
@@ -223,37 +223,44 @@
           <string>Miscellaneous</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_groupBox_miscellaneous">
-          <item row="0" column="0">
+          <item row="1" column="0">
```

Comment:
These changes are, as usual for the Qt Designer plug-in/utility, scrambling the order of the items in a `QGridLayout` that has been touched by that tool. If possible, please put them back in ascending row (first) then column order to try and keep the `git` / `diff` changes in this file as simple/clean as possible...

:pray: 

Also, if you have been adding any controls they will automatically be put to the bottom of the tab oder (how the focus moves forward/back each time \<Tab>/\<Shift>+\<Tab> is used) so you may want to edit that {by selecting the "Edit tab order" tab} in the tool:
![image](https://user-images.githubusercontent.com/6163092/134826413-80b2109e-ef6b-4fc7-9284-25b10c90a918.png)

You can then right-click on the item and select "Tab Order List..." to adjust the position of the item in the list...
![image](https://user-images.githubusercontent.com/6163092/134826465-2a37c6b2-845d-4660-b1f7-93ec8220a892.png)

... by selecting the item(s) and using the up and down arrow buttons on the right to move them:
![image](https://user-images.githubusercontent.com/6163092/134826483-b1953aa9-a2b5-4b76-8ecf-811ae2f9bbac.png)

Reply:
The tab order is good to get right, but don't worry about the ordering in the XML by hand, it adds some but not enough value for the effort involved 😕